book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+  end
```

Comment:
- [OPTIONAL] The add_rental method should include person and date like so:
 ` def add_rental(person, date)`
   ` Rental.new(date, self, person)`
`  end`

Please kindly fix that.